core/params.go

```diff
@@ -2,13 +2,15 @@ package core
 
 import (
 	"math/big"
+
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
 )
 
 // Params stores all the parameters which are maintained on-chain and keeps updating
 // them whenever they change on-chain
 type Params struct {
-	DBModel
-
+	ID string `json:"-" gorm:"primary_key;size:100;default:'6ba7b810-9dad-11d1-80b4-00c04fd430c8'"`
```

Comment:
Can you elaborate what is `6ba7b810-9dad-11d1-80b4-00c04fd430c8`?

Reply:
Yeah, it's the namespace for `DNS`the lib set this up.

ctrl+f on https://tools.ietf.org/html/rfc4122.html#section-4.3